gillespy2/solvers/numpy/ode_solver.py

```diff
@@ -88,26 +88,41 @@ def run(self=None, model=None, t=None, number_of_trajectories=1, increment=None,
             integrator_options={}, live_output=None, live_output_options={}, timeout=None, resume=None, **kwargs):
         """
         :param model: gillespy2.model class object
```

Comment:
Add `(deprecated)` to the description of the model are to indicate that this arg is deprecated